llm_studio/python_configs/text_dpo_modeling_config.py

```diff
@@ -27,7 +27,6 @@ class ConfigDPODataset(ConfigNLPCausalLMDataset):
     # Always have full chat history.
     # Chosen/Rejected prompt are only at the end of a conversation.
     limit_chained_samples: bool = True
```

Comment:
With the new setting, would it be possible to make this customizable again?
Say, only last answers or all answers (but only chained up to the one in question)

Reply:
not sure I fully get it, could you please elaborate?

Reply:
Realized that the PR solves it already. In the end there are two options as in standard Causal Language Modeling:
- calculate loss on all answers 
- calculate loss on just the last answer